app/presenters/content_item_presenter.rb

```diff
@@ -31,6 +29,22 @@ def content_id
     content_item.parsed_content["content_id"]
   end
 
+  def taxons
```

Comment:
Breadcrumbs aren't included on every format.
We use a mixin pattern to include features when they're needed.
Should these methods be in a mixin and included on just the formats that need them?

This will prevent the single content_item_presenter from getting too large.